flashlight/pkg/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.10)
+
+set(FL_PKG_VISION_DIR ${FL_PKG_DIR}/vision)
```

Comment:
Also ignore if handled later, but we should gate building these packages based on which apps the user wants to build

Reply:
handled later.